src/redux/actions/axiosconfig.ts

```diff
@@ -8,8 +8,7 @@ config.interceptors.request.use(
   (config) => {
     const token = localStorage.getItem("access_token");
     if (token) {
-      config.headers = config.headers || {};
-      config.headers["Authorization"] = `Bearer ${token}`;
+      config.headers["Authorization"] = `${token}`;
```

Comment:
Are you sure this is not going to cause an issue?

Reply:
I can't login as an admin

Reply:
I signed up that account of admin again since it was deleted when they run seed, you can check again.
it is working as it should.

Reply:
> Are you sure this is not going to cause an issue?

Yes That is the main fix of this PR since the last PR that was merged on develop messed this up.